lib/client/client.go

```diff
@@ -1154,6 +1169,13 @@ func (c *NodeClient) handleGlobalRequests(ctx context.Context, requestCh <-chan
 			}
 
 			switch r.Type {
+			case teleport.MFAPresenceRequest:
+				if c.OnMFA == nil {
+					log.Warn("Received MFA presence request, but no callback was provided.")
```

Comment:
No. This is an error that no handler for running the presence task was found. This is set up earlier but I added this if the code is changed in the future to catch any errors. This doesn't allow bypass as the presence verification is made by the target node regardless of the presence task. The result of this is that the client will be kicked the node won't detect any presence updates.

Reply:
I've reasoned this is an earlier review comment last week but this callback system looks like it does due to the architecture of NodeClient.

Reply:
In addition, as I've said before, not waiting for the first presence verification event is fine since the client already has demonstrated the presence by the MFA requirement for issuing the certs.